cmake/Windows/Contrib/Inetc/inetc.vcxproj

```diff
@@ -42,42 +42,42 @@
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release Unicode|Win32'" Label="Configuration">
     <ConfigurationType>DynamicLibrary</ConfigurationType>
-    <PlatformToolset>v110</PlatformToolset>
+    <PlatformToolset>v142</PlatformToolset>

```

Comment:
Anyone knows why the toolset is higher here? is this loaded and automatically updated by Visual Studio?

Reply:
Probably by VS. Honestly I do not know what this number means.. Should I keep v110?

Reply:
yeah probably better to just use the original file.